src/main/kotlin/io/snyk/plugin/ui/settings/ScanTypesPanel.kt

```diff
@@ -142,6 +142,7 @@ class ScanTypesPanel(
             }
         }
     }.apply {
+        name = "scanTypesPanel"
```

Comment:
what's the purpose of that naming? debug only? Seems we don't use it anywhere...

Reply:
Makes it easier to find the panel in testing and debugging, so naming them is always good :)